app/javascript/packs/components/AddPopup.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { Modal, Button, FormGroup, FormLabel, FormControl } from 'react-bootstrap';
+
+import { fetch } from './Fetch';
+
+export default class AddPopup extends Component {
+  state = {
+    name: '',
+    description: '',
+    assignee: {
```

Comment:
I don't see a place where you change the `assignee`. And if you don't you shouldn't place this in the state.